lochness/keyring/__init__.py

```diff
@@ -40,11 +40,11 @@ def box_api_token(Lochness, key):
     '''get box api token from keyring'''
     if key not in Lochness['keyring']:
         raise KeyringError('\'{0}\' not in keyring'.format(key))
-    if 'API_TOKEN' not in Lochness['keyring'][key]:
-        raise KeyringError('\'API_TOKEN\' not in {0}'.format(key))
+    if 'USER_ID' not in Lochness['keyring'][key]:
```

Comment:
BOX related changes have already been reviewed in https://github.com/AMP-SCZ/lochness/pull/50/files